File: Problems/C++/CodeForce/Codeforce-round-861/C_Unlucky_Numbers.py
t = int(input())

for i in range(t):
    l, r = map(int, input().split())
    min_diff = 9
    num = l
    for j in range(l, r+1):
        diff = int(max(str(j))) - int(min(str(j)))
        if diff < min_diff:
            min_diff = diff
            num = j
            if min_diff == 0:
                break
    print(num)

File: Problems/C++/CodeForce/Codeforce-round-861/C_Unlucky_Numbers.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int t;
    cin >> t;
    for (int i = 0; i < t; i++) {
        int l, r;
        cin >> l >> r;
        int min_diff = 9;
        int num = l;
        for (int j = l; j <= r; j++) {
            string s = to_string(j);
            int diff = max(s) - min(s);
            if (diff < min_diff) {
                min_diff = diff;
                num = j;
                if (min_diff == 0) {
                    break;
                }
            }
        }
        cout << num << endl;
    }
    return 0;
}
